source/install-by-copying-files/index.html.md.erb

```diff
@@ -0,0 +1,73 @@
+---
+title: Install by copying files
+weight: 10
+---
+
+# Install by copying files
+
+You can install GOV.UK Frontend by copying our CSS and JavaScript files into your project.
+
+<%= warning_text('We recommend <a href="/installing-with-npm/">installing with Node.js package manager (npm)</a> instead.
+') %>
+
+## What you cannot do
+
+If you install this way, you will not be able to:
+
+- change settings, for example you will not be able to override colours or set your own font
+- use the Nunjucks macros from the [Design System website](https://design-system.service.gov.uk/)
+- import an individual component’s CSS or JavaScript
+- use GOV.UK Frontend's colours or mixins in your custom code
+
+## Copy the files
+
+1. Download the [`dist` folder and its subfolders](https://github.com/alphagov/govuk-frontend/tree/master/dist) from the GOV.UK Frontend GitHub repo.
+2. Copy the 2 `.css` files to `<YOUR-APP>/stylesheets`.
+3. Copy the `.js` file to `<YOUR-APP>/javascript`.
+4. Copy the `assets` folder to the root of your project’s public folder, so that for example `<YOUR-SITE-URL>/assets/govuk-logotype-crown.png` will work in your users’ browsers.
+
+## Check it works
+
+1. Create a page in your project using the following HTML:
+
+    ```html
+    <!DOCTYPE html>
+    <html lang="en" class="govuk-template ">
+      <head>
+        <title>Example if you’ve installed GOV.UK Frontend by copying files</title>
+        <meta name="viewport" content="width=device-width, initial-scale=1, viewport-fit=cover">
+        <!--[if !IE 8]><!-->
+          <link rel="stylesheet" href="/stylesheets/govuk-frontend-<VERSION-NUMBER>.min.css">
+        <!--<![endif]-->
+        <!--[if IE 8]>
+          <link rel="stylesheet" href="/stylesheets/govuk-frontend-ie8-<VERSION-NUMBER>.min.css">
+        <![endif]-->
+      </head>
+      <body class="govuk-template__body ">
+        <script>
+          document.body.className = ((document.body.className) ? document.body.className + ' js-enabled' : 'js-enabled');
+        </script>
+        <!-- component HTML -->
+        <script src="/javascript/govuk-frontend-<VERSION-NUMBER>.min.js"></script>
+        <script>
+          window.GOVUKFrontend.initAll()
+        </script>
+      </body>
+    </html>
+    ```
+
+    In your live application, you should use the [page template](https://design-system.service.gov.uk/styles/page-template/) instead.
+
+2. Replace `<VERSION-NUMBER>` so the 3 filenames match the files you copied from GOV.UK Frontend's GitHub repo.
+
+3. Go to the [example accordion component](https://design-system.service.gov.uk/components/accordion/#accordion-example) on the Design System website, then copy the HTML.
+
+4. Replace `<!-- component HTML -->` with the accordion HTML you copied.
+
+5. Run your application and check it works the same way as the Design System accordion example, by selecting the buttons and checking the accordion shows and hides sections.
```

Comment:
Because the HTML doesn't contain any of the other 'boilerplate' layout, you end up with something that looks like this:

![image](https://user-images.githubusercontent.com/121939/81060144-c43b1b00-8ec9-11ea-974e-3dc27702eae8.png)

Do we think that's OK? Do we need to do anything to set expectations about what they should see, or make it clearer what the next steps would be to get something that looks more like a 'complete' page?

Reply:
Ah good point - I think if we use your suggestion of making it clearer what the next steps are, it should probably be ok.

I'll add the same line that we have at the end of 'Getting started': "You can now get the full code for page layouts and other components from the [Design System website](https://design-system.service.gov.uk/)."

Reply:
I've got some doubts about this as well, we say in step 5
> Run your application and check it works the same way as the Design System accordion example

That to me sounds like it should "look" the same as well. I would probably think I've done something wrong if I saw a full page version of the accordion, it looks broken to me 🤔 

I wonder if we can try to manage expectations by telling them more explicitly what they should expect to see.

Reply:
Could we link them to https://design-system.service.gov.uk/components/accordion/default/index.html to compare to that instead maybe?

Reply:
@m-green Yes that's a good idea 👍